models/parrot_neuron.h

```diff
@@ -145,9 +149,20 @@ parrot_neuron::send_test_event( Node& target, rport receptor_type, synindex, boo
 inline port
 parrot_neuron::handles_test_event( SpikeEvent&, rport receptor_type )
 {
-  if ( receptor_type != 0 )
+  // Allow connections to port 0 (spikes to be repeated)
+  // and port 1 (spikes to be ignored).
+  if ( receptor_type == 0 )
```

Comment:
I agree, but tried to stick to the existing coding style (see https://github.com/nest/nest-simulator/blob/master/models/iaf_psc_exp.h#L354). Will change this here.